main.py

```
@@ -10571,6 +10586,36 @@ async def fetch_ranked(query: str, order_by: str, ahead_of: str, *args) -> list:
                     message.guild.id,
                 )
                 final_value = "fish_caught"
+            case "Aura":
+                unit = "auras"
+                if specific_cat in AURA_ORDER:
+                    _a = specific_cat
+                    _count_expr = RawSQL(f"(SELECT COUNT(*) FROM unnest(cat_auras) v WHERE v = '{_a}') AS aura_count")
+                    result = await Profile.collect_limit(
+                        ["user_id", _count_expr],
+                        f"guild_id = $1 AND (SELECT COUNT(*) FROM unnest(cat_auras) v WHERE v = '{_a}') > 0 ORDER BY aura_count DESC",
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)